src/components/Root.tsx

```diff
@@ -32,7 +32,10 @@ export default forwardRef<HTMLDivElement, Props>(function Root(
   }, []);
 
   return (
-    <div ref={ref} className={ROOT_CLASS}>
+    <div
+      ref={ref}
+      className={ROOT_CLASS + ' h-screen max-h-full min-h-full overflow-auto'}
```

Comment:
What dfoes h-screen do here? DOes it work when we set both height and max height?

Reply:
Also, how about the width?

Reply:
I am wondering, since maybe doing `flex-col` automatically fills the whole available width?